README.md

```diff
@@ -20,10 +20,20 @@ questions.
 
 This repo contains the source used to generate a static site using [Nikola](https://getnikola.com/).
 
-Nikola installation and documentation can be found [here](https://getnikola.com/getting-started.html#install).
-
 Configuration file for the site is ``conf.py``.
 
+### Requirements
+
+- Visit [Nikola installation documentation](https://getnikola.com/getting-started.html#install).
+- Install `sass` to compile scss files to css.
+
+You can install `sass` as described in the [Sass install docs](https://sass-lang.com/install) or:
+
```

Comment:
I would mention here that these are instructions specific for [Fedora](https://getfedora.org/).

Reply:
Thanks I've updated to point that out.